test/ussd_public.test.js

```diff
@@ -271,6 +271,32 @@ describe("familyconnect health worker app", function() {
                     })
                     .run();
             });
+            it("to state_last_period_month (recognised, optedout user)", function() {
+                return tester
+                    .setup.user.addr('0720000888')
+                    .inputs(
+                        {session_event: 'new'}  // dial in
+                    )
+                    .check.interaction({
+                        state: 'state_last_period_month',
+                        reply: [
+                            "What month did you start your last period?",
+                            "1. Apr 15",
+                            "2. Mar 15",
+                            "3. Feb 15",
+                            "4. Jan 15",
+                            "5. Dec 14",
+                            "6. Nov 14",
+                            "7. Oct 14",
+                            "8. Sep 14",
+                            "9. Aug 14"
```

Comment:
This date format has changed in the base branch, which is why you're getting test errors.